compose.js

```diff
@@ -88,12 +92,10 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('configuration', assign);
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
-    dstDescriptor.initializers = srcDescriptor.initializers.reduce((result, init) => {
-      if (isFunction(init) && result.indexOf(init) < 0) {
-        result.push(init);
-      }
-      return result;
-    }, Array.isArray(dstDescriptor.initializers) ? dstDescriptor.initializers : []);
+    const initializers = (dstDescriptor.initializers || [])
```

Comment:
I cannot imagine why would tests actually fail since only `compose` is exported and this is completely internal. I would even say to move that `uniq` call to `createFactory`. We are merging here, so why not to keep logic here as minimal as possible so it's more readable?